help_to_heat/frontdoor/views.py

```diff
@@ -644,10 +662,22 @@ def save_post_data(self, data, session_id, page_name):
 @register_page(address_select_page)
 class AddressSelectView(PageView):
     def build_extra_context(self, request, session_id, page_name, data, is_change_page):
+        session_data = interface.api.session.get_session(session_id)
+        show_epc_update_details = session_data.get(country_field) in [country_field_wales, country_field_england]
+
         data = interface.api.session.get_answer(session_id, address_page)
         building_name_or_number = data[address_building_name_or_number_field]
         postcode = data[address_postcode_field]
         addresses = interface.api.address.find_addresses(building_name_or_number, postcode)
+
+        current_month_start = datetime.now().replace(day=1)
+        month_name = month_names[current_month_start.month - 1]
+        current_month_start = current_month_start.strftime("%-d") + " " + month_name
+
+        next_month_start = (datetime.now() + relativedelta(months=+1)).replace(day=1)
+        month_name = month_names[next_month_start.month - 1]
+        next_month_start = next_month_start.strftime("%-d") + " " + month_name
```

Comment:
I think this can be simpler if we hardcode the 1 in each case? We're effectively doing that already!